Scientist4JCore/src/main/java/com/github/rawls238/scientist4j/Experiment.java

```diff
@@ -25,29 +28,29 @@
     private final Counter totalCount;
 
     public Experiment() {
-      this("Experiment");
+        this("Experiment");
     }
 
     public Experiment(String name) {
         this(name, false);
     }
 
     public Experiment(String name, Map<String, Object> context) {
-      this(name, context, false);
+        this(name, context, false);
     }
 
     public Experiment(String name, boolean raiseOnMismatch) {
-      this(name, new HashMap(), raiseOnMismatch);
+        this(name, new HashMap<>(), raiseOnMismatch);
     }
 
     public Experiment(String name, Map<String, Object> context, boolean raiseOnMismatch) {
-      this.name = name;
-      this.context = context;
-      this.raiseOnMismatch = raiseOnMismatch;
-      controlTimer = metrics.timer(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "control"));
-      candidateTimer = metrics.timer(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "candidate"));
-      mismatchCount = metrics.counter(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "mismatch"));
-      totalCount = metrics.counter(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "total"));
+        this.name = name;
```

Comment:
I am pretty sure 4 spaces (preferred) or Tab is the standard.  I haven't been anywhere that uses 2 spaces for java code.  I will update if that makes your eyeballs happier.  (Although I am pretty sure if would take you less time to fix after merging than me going through updating and rebasing, squashing etc)